### PR TITLE
Cleaned schemas, fixed tests

### PR DESCRIPTION
**Pull request by kprzerwa**

### CODE REVIEW

File: cds_dojson/schemas/records/books/book/base-v0.0.1.json [hunk 1]
```diff
@@ -0,0 +1,587 @@
+{
+  "decription": "A record in the books, proceedings, standards, theses and reports.", 
+  "title": "CDS Base Record Schema v1.0.0", 
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "required": [
+    "document_type", 
+    "titles",
```
Comment (topless) [inline cds_dojson/schemas/records/books/book/base-v0.0.1.json]: I believe this is `title` not titles

Comment (topless) [reply cds_dojson/schemas/records/books/book/base-v0.0.1.json]: I noticed there is `titles` further down.. so you can continue with your life!

File: cds_dojson/schemas/records/books/book/elements/language_code.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    "wo", 
+    "xh", 
+    "yi", 
+    "yo", 
+    "za", 
+    "zh", 
+    "zu"
+  ], 
+  "type": "string"
+}
```
Comment (topless) [inline cds_dojson/schemas/records/books/book/elements/language_code.json]: Prettier works with json files too, run a code format on them.

File: cds_dojson/schemas/records/books/ymls/elements/language_code.yml [hunk 3]
```diff
@@ -3,190 +3,190 @@ description: |-
...
-- vo
-- wa
-- wo
-- xh
-- yi
-- yo
-- za
-- zh
-- zu
+    - aa
```
Comment (topless) [inline cds_dojson/schemas/records/books/ymls/elements/language_code.yml]: A general comment, for these and all the enums before and after, I usually select them all and `cmd` + `shift` + `p` in vs code and I select sort. Once they are sorted, my ocds go away and we can also spot instantly if there are any duplicates, which was the case in the languages in the past.

File: cds_dojson/schemas/records/books/book/elements/language_code.json (hunk 2 above)
Comment (topless) [reply cds_dojson/schemas/records/books/book/elements/language_code.json]: It doesn't show but I am referring in ending with an empty line every file.

File: cds_dojson/schemas/records/books/ymls/elements/language_code.yml (hunk 3 above)
Comment (kprzerwa) [reply cds_dojson/schemas/records/books/ymls/elements/language_code.yml]: You can sleep calmly from now :)

**Comment by kprzerwa** [comment]
all suggestions are fixed

**Pull request merged**